.github/CODEOWNERS

```
@@ -0,0 +1,33 @@
+# Automatically request reviews from maintainers
+
+# Package
+/pybamm/discretisations/ @martinjrobins @rtimms @tinosulzer
+/pybamm/experiment/ @brosaplanella @martinjrobins @rtimms @tinosulzer @TomTranter
+/pybamm/expression_tree/ @martinjrobins @rtimms @tinosulzer
+/pybamm/geometry/ @martinjrobins @rtimms @tinosulzer
+/pybamm/input/ @brosaplanella @DrSOKane @rtimms @tinosulzer @TomTranter @kratman
+/pybamm/meshes/ @martinjrobins @rtimms @tinosulzer @rtimms
+/pybamm/models/ @brosaplanella @DrSOKane @rtimms @tinosulzer @TomTranter @rtimms
+/pybamm/parameters/ @brosaplanella @DrSOKane @rtimms @tinosulzer @TomTranter @rtimms  @kratman
+/pybamm/plotting/ @martinjrobins @rtimms @Saransh-cpp @tinosulzer @rtimms  @kratman
```

```suggestion
/pybamm/plotting/ @martinjrobins @rtimms @Saransh-cpp @tinosulzer @rtimms  @kratman @agriyakhetarpal 
```
I am happy to learn and maintain the plotting functionality too as we go along